apps/src/dance/DanceAiModal.tsx

```diff
@@ -143,7 +147,13 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
   };
 
   const startAi = async (value: string) => {
-    const responseJsonString = await doAi(value);
+    let responseJsonString: any;
+    // Default to using cached response, otherwise contact OpenAI directly
+    if (queryParams('ai-model') === 'llm') {
+      responseJsonString = await doAi;
```

Comment:
Shouldn't we be passing `value` to `doAi`?  We need to test that this code path still works.

Reply:
Good catch, I'll make sure to pass in the value